.gitignore

```diff
@@ -148,6 +148,57 @@ zz_generated.openapi.go
 /barbican-kms-plugin
 /magnum-auto-healer
 /manila-csi-plugin
+/barbican-kms-plugin-amd64
+/barbican-kms-plugin-arm32v7
+/barbican-kms-plugin-arm64v8
+/barbican-kms-plugin-ppc64le
+/barbican-kms-plugin-s390x
+/cinder-csi-plugin-amd64
+/cinder-csi-plugin-arm32v7
+/cinder-csi-plugin-arm64v8
+/cinder-csi-plugin-ppc64le
+/cinder-csi-plugin-s390x
+/cinder-flex-volume-driver-amd64
+/cinder-flex-volume-driver-arm32v7
+/cinder-flex-volume-driver-arm64v8
+/cinder-flex-volume-driver-ppc64le
+/cinder-flex-volume-driver-s390x
+/cinder-provisioner-amd64
+/cinder-provisioner-arm32v7
+/cinder-provisioner-arm64v8
+/cinder-provisioner-ppc64le
+/cinder-provisioner-s390x
+/client-keystone-auth-amd64
+/k8s-keystone-auth-amd64
+/k8s-keystone-auth-arm32v7
+/k8s-keystone-auth-arm64v8
+/k8s-keystone-auth-ppc64le
+/k8s-keystone-auth-s390x
+/magnum-auto-healer-amd64
+/magnum-auto-healer-arm32v7
+/magnum-auto-healer-arm64v8
+/magnum-auto-healer-ppc64le
+/magnum-auto-healer-s390x
+/manila-csi-plugin-amd64
+/manila-csi-plugin-arm32v7
+/manila-csi-plugin-arm64v8
+/manila-csi-plugin-ppc64le
+/manila-csi-plugin-s390x
+/manila-provisioner-amd64
+/manila-provisioner-arm32v7
+/manila-provisioner-arm64v8
+/manila-provisioner-ppc64le
+/manila-provisioner-s390x
+/octavia-ingress-controller-amd64
+/octavia-ingress-controller-arm32v7
+/octavia-ingress-controller-arm64v8
+/octavia-ingress-controller-ppc64le
+/octavia-ingress-controller-s390x
+/openstack-cloud-controller-manager-amd64
+/openstack-cloud-controller-manager-arm32v7
+/openstack-cloud-controller-manager-arm64v8
+/openstack-cloud-controller-manager-ppc64le
+/openstack-cloud-controller-manager-s390x
```

Comment:
Could be simplified using regex, but this shouldn't block the merge.

Reply:
IMO we can simplify it once all details are settled (includes the following change in test jobs under openlib repo)

Reply:
resolved